backup_and_restore/application_backup_and_restore/installing/configuring-backup-restore-pvc-datamover.adoc

```diff
@@ -0,0 +1,21 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="configuring-backup-restore-pvc-datamover"]
+= Configuring `backupPVC` and `restorePVC` for Data Mover
+include::_attributes/common-attributes.adoc[]
+:context: configuring-backup-restore-pvc-datamover
+
+toc::[]
+
+A `backupPVC` is an intermediate `PersistentVolumeClaim` (PVC) to access data during the data movement backup operation. For some storage classes, such as, CephFS, creating a read-only volume from a snapshot results in faster Data Mover backups.
```

Comment:
> For some storage classes, such as, CephFS, creating a read-only volume from a snapshot results in faster Data Mover backups.

This could be my ignorance, but I am not clear on what this sentence means or how it relates to the assembly. Is the read-only volume what we are working on, the backup PVC is a read-only volume?